### PR TITLE
Deprecation Notice

### PR DESCRIPTION
**Pull request by juanpedromoreno**
Adds a deprecation notice before the project is archived.

Thanks for your services

### CODE REVIEW

**Review by BenFradet — APPROVED** [review]
:tada:

**Comment by cb372** [comment]
EQ_6P1GXYAYup9N

**Review by fedefernandez — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,9 +4,13 @@
...
 [comment]: # (End Badges)
-# sbt-org-policies
+# sbt-org-policies (ABANDONED)
 
-**sbt-org-policies** is just an SBT plugin that enforces you with some organization policies and settings across all your projects.
+### DEPRECATION NOTICE
+
+**sbt-org-policies** was an SBT plugin that used to enforce some organization policies and sharing settings across all the Scala projects. However, with Github actions, as well as other fantastic sbt plugins like sbt-ci-release, sbt-mdoc, sbt-github, ... you can achieve the same in a more natural, and maintainable way. We recommend using that approach.
+
+47 Degrees has therefore discontinued the development and maintenance of this library. The source code is left here for those interesting in studying it.
```
Comment (franciscodr) [inline README.md:13]:
```suggestion
47 Degrees has therefore discontinued the development and maintenance of this library. The source code is left here for those interested in studying it.
```

**Pull request merged**